modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/AbstractCreateIndexCommand.java

```diff
@@ -49,10 +50,22 @@ public abstract class AbstractCreateIndexCommand extends AbstractIndexCommand {
 
     protected final List<String> columns;
 
+    protected final CatalogIndexStatus status;
```

Comment:
Looks like a field that is not needed, a new logical `boolean` with documentation is enough.